R/r.R

```diff
@@ -38,30 +38,18 @@
 #'
 #' The [rename_files()] function can also be helpful.
 #'
-#' @param name Either a name without extension, or `NULL` to create the
-#'   paired file based on currently open file in the script editor. If
-#'   the `R/` file is open, `use_test()` will create/open the corresponding
-#'   test file; if the test file is open, `use_r()` will create/open the
-#'   corresponding `R/` file.
+#' @param name Either a string giving a file name (without directory) or
+#'   `NULL` to take the name from the currently open file in RStudio.
 #' @inheritParams edit_file
 #' @seealso The [testing](https://r-pkgs.org/tests.html) and
 #'   [R code](https://r-pkgs.org/r.html) chapters of
 #'   [R Packages](https://r-pkgs.org).
 #' @export
 use_r <- function(name = NULL, open = rlang::is_interactive()) {
-  check_not_empty_file_name(name)
```

Comment:
I inlined `check_not_empty_filename()` into `check_file_name()` and moved that into `compute_name()`.